lib/main.coffee

```diff
@@ -20,6 +25,10 @@ module.exports =
     @subscriptions.add atom.config.observe 'linter-shellcheck.enableNotice',
       (enableNotice) =>
         @enableNotice = enableNotice
+    @subscriptions.add atom.config.observe 'linter-shellcheck.enableFixPath',
+      (enableFixPath) ->
+        # can not reverse fix-path once loaded
+        require('fix-path')() if enableFixPath
```

Comment:
If more packages do the same, it would add duplicate paths to the global. We shouldn't be altering globals when we don't have to, see https://github.com/AtomLinter/atom-linter/issues/62#issuecomment-163041242